plugins/chat/assets/javascripts/discourse/modifiers/chat/track-message.js

```diff
@@ -0,0 +1,39 @@
+import Modifier from "ember-modifier";
+import { registerDestructor } from "@ember/destroyable";
+import { bind } from "discourse-common/utils/decorators";
+
+export default class ChatTrackMessage extends Modifier {
```

Comment:
Why is this needed when there is already `track-message-visibility.js`, they both do the same thing from what I can see? I am probably wrong, just curious